### ISSUE TITLE
how can i use this in vue-cli

### ISSUE THREAD

**Issue by kavanLi** [alvarotrigo/vue-fullpage.js#11]
![image](https://user-images.githubusercontent.com/33696428/38132120-50e3ffee-343c-11e8-90ee-41ba328887cd.png)

![image](https://user-images.githubusercontent.com/33696428/38132131-5cd7300a-343c-11e8-92e4-557b15176014.png)

I have install vue-fullpage through npm install --save vue-fullpage.js.

and got error, please see details from above image

**Comment by ebisbe** [comment]
@kavanLi It happened the same to me. I'm using this:
```
import FullPage from 'vue-fullpage.js'
```

**Comment by noobling** [comment]
Are the docs just out of date? Whats going on?

**Comment by LazeBear** [comment]
Here is how I solved it:
beside vue-fullpage.js, you also need to install fullpage.js
```npm install fullpage.js --save```
then in your vue file
```
import 'fullpage.js';
import FullPage from 'vue-fullpage.js/src/FullPage.vue';
import fullPageMixin from 'vue-fullpage.js/src/fullPageMixin';
```
The rest just follow the basic example on readme

**Comment by noobling** [comment]
GG

**Comment by alvarotrigo** [comment]
This should be fixed in the latest version 0.0.2
